pages/common/omf.md

```diff
@@ -0,0 +1,28 @@
+# omf
+
+> Oh My Fish, the Fishshell Framework.
+> Install packages to extend and modify the fish shell.
+
+- Install one or more packages:
```

Comment:
How are multiple packages specified ? Separated by space ? We should mention that.

Reply:
Yeh, they are space separated according to the docs. :+1: